app/Jobs/SendEmptyWikibaseNotificationsJob.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Notifications\EmptyWikibaseNotification;
+use App\Wiki;
+use Carbon\CarbonImmutable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Support\Facades\Log;
+
+class SendEmptyWikibaseNotificationsJob extends Job implements ShouldBeUnique
+{
+    public function handle (): void
+    {
+        $allWikis = Wiki::all();
+        foreach ($allWikis as $wiki) {
+            try {
+                $this->sendEmptyWikibaseNotification($wiki);
+            } catch (\Exception $exception) {
+                Log::error(
+                    'Failure polling wiki '.$wiki->getAttribute('domain').' for sitestats: '.$exception->getMessage()
+                );
+            }
+        }
+    }
+
+    public function sendEmptyWikibaseNotification (Wiki $wiki): void
```

Comment:
Very nitpicky: the method name is a bit misleading. It sounds as if this will always send a notification, whereas in reality it might skip a lot of wikis. Maybe this can be broken down in two methods anyways: 1. check if a wiki is eligible, 2. send notification

Reply:
COuld you give me a name suggestion?

Reply:
I'd break it down into two methods:

```php
$isEligible = $this->checkIfWikiIsEligible($wiki);
if ($isEligible) {
    $this->sendEmptyWikibaseNotification($wiki);
}
```